.github/workflows/ci-tests.yml

```diff
@@ -48,6 +64,8 @@ jobs:
           pip install setuptools
           pip install google
           pip install protobuf
+          
+          go install golang.org/x/tools/cmd/goimports@latest
```

Comment:
This has a make target (make deps-install or something)

Reply:
Yeah, it is absolute, was part of the debugging. Will remove.